src/components/Modals/PublishCollectionModal/PublishCollectionModal.tsx

```diff
@@ -62,93 +79,165 @@ export default class PublishCollectionModal extends React.PureComponent<Props, S
-        {t('publish_collection_modal.second_paragraph')}
-        <div className="divider"></div>
-        {t('publish_collection_modal.third_paragraph')}{' '}
-        <a href="https://docs.decentraland.org/wearables/publishing-wearables" rel="noopener noreferrer" target="_blank">
-          {t('global.learn_more')}
-        </a>
-        <Button primary fluid onClick={this.handlePublish} loading={isLoading}>
-          {t('publish_collection_modal.publish')}
-        </Button>
+        <ModalNavigation title={t('publish_collection_modal.title_tos')} onClose={onClose} />
+        <Modal.Content className="second-step">
+          {t('publish_collection_modal.first_paragraph')}
+          <div className="divider"></div>
+          {t('publish_collection_modal.second_paragraph')}
+          <div className="divider"></div>
+          {t('publish_collection_modal.third_paragraph')}{' '}
+          <a href="https://docs.decentraland.org/wearables/publishing-wearables" rel="noopener noreferrer" target="_blank">
+            {t('global.learn_more')}
+          </a>
+          <Button primary fluid onClick={this.handleNextStep}>
+            {t('global.next')}
+          </Button>
+        </Modal.Content>
+      </>
+    )
+  }
+
+  renderStep = () => {
+    const { step } = this.state
+    switch (step) {
+      case 1:
+        return this.renderFirstStep()
+      case 2:
+        return this.renderSecondStep()
+      case 3:
+        return this.renderThridStep()
+      default:
+        throw new Error('Step not found')
+    }
+  }
+
+  renderThridStep = () => {
+    const { isLoading, onClose } = this.props
+    const { email, emailFocus } = this.state
+    const hasValidEmail = emailRegex.test(email ?? '')
+    const showEmailError = !hasValidEmail && !emailFocus && email !== undefined && email !== ''
+
+    return (
+      <>
+        <ModalNavigation title={t('publish_collection_modal.title_tos')} onClose={onClose} />
+        <Modal.Content className="third-step">
+          <div className="tos">
+            <p>{t('publish_collection_modal.tos_title')}</p>
+            <p>
+              <T
+                id="publish_collection_modal.tos_first_condition"
+                values={{
+                  terms_of_use: (
+                    <a href="https://decentraland.org/terms/" rel="noopener noreferrer" target="_blank">
+                      {t('publish_collection_modal.terms_of_use')}
+                    </a>
+                  ),
+                  content_policy: (
+                    <a href="https://decentraland.org/content/" rel="noopener noreferrer" target="_blank">
+                      {t('publish_collection_modal.content_policy')}
+                    </a>
+                  )
+                }}
+              />
+            </p>
+            <p>{t('publish_collection_modal.tos_second_condition')}</p>
+            <p>{t('publish_collection_modal.tos_third_condition')}</p>
+          </div>
+          <Field
```

Comment:
I think we could do two things to improve the UX of this field:
- We can delete the email validation and just use a Field with type email
- We could add a Form (from dcl-ui) so that pressing Enter (return key) on the field submits the form

if any of these two things become a pain to do, the current implementation is fine!